tests/data_import/test_ze_aliases.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+from django.core.exceptions import ValidationError
+
+from data_import.models import RespondingAgencyAlias
+from payroll.models import EmployerAlias
+
+
+@pytest.mark.django_db
+def test_responding_agency_preferred_alias(responding_agency):
+    agency = responding_agency.build()
+
+    old_alias = RespondingAgencyAlias.objects.create(responding_agency=agency, name='a_rose', preferred=True)
+
+    new_alias = RespondingAgencyAlias.objects.create(responding_agency=agency, name='by_any_other_name', preferred=True)
+
+    old_alias_again = RespondingAgencyAlias.objects.get(id=old_alias.id)
```

Comment:
Question I missed the first time: Why do you need to query for the old alias again? Can you use `old_alias`?

Reply:
When I did this, it looked like that variable was storing information from the creation, but not getting the updated information necessary to verify that the `preferred` value had changed. 
Therefore, I went ahead and got that alias again to make sure I had the updated information

Reply:
Ah, right! You can use [`obj.refresh_from_db()`](https://docs.djangoproject.com/en/2.2/ref/models/instances/#django.db.models.Model.refresh_from_db) for that!